src/interfaces/vault.ts

```diff
@@ -527,6 +532,10 @@ export class VaultInterface<T extends ChainId> extends ServiceInterface<T> {
     dynamic.metadata.migrationAvailable = dynamic.metadata.migrationAvailable || overrides.migrationAvailable || false;
   }
 
+  private shouldUsePartnerService(vault: string): boolean {
+    return !!this.yearn.services.partner && this.yearn.services.partner.isAllowed(vault);
+  }
```

Comment:
This is in 2 places, could we have this as a util function or that doesn't make sense in this context, used here as well https://github.com/yearn/yearn-sdk/pull/224/files#diff-c02a4a6d0b4ad24cba60f4ad229198c6d149c626b2780875abcf65f2d6ea04deR487

Same feedback, could be simplified with the elvis operator to
```suggestion
  private shouldUsePartnerService(vault: string): boolean {
    return this.yearn.services.partner?.isAllowed(vault);
  }
```

Reply:
Fixed on 03bf09134a3c47a2521dad867c47195f8c52bfca